src/components/SearchFilterMobile.js

```diff
@@ -5,6 +5,7 @@ import { useRouter } from 'next/router';
 import React from 'react';
 import Link from './Link';
 import Filter from './common/Filter';
+import { useEmbed } from '../hooks/globalHooks';
```

Comment:
@okdv don't need to worry about the stuff related to search/filter, we are not using it, and we will re-implmenent the logic in the future.